app/kubemci/pkg/validations/validations.go

```diff
@@ -89,3 +91,69 @@ func nodePortSameInAllClusters(backend v1beta1.IngressBackend, namespace string,
 	}
 	return nil
 }
+
+func ServerVersionsNewEnough(clients map[string]kubeclient.Interface) (bool, error) {
```

Comment:
Does this have to be public?
Call this from the existing public Validate method?

Reply:
I think it needs to be public. validations.Validate is called from loadbalancersyncer.go. ServerVersionsNewEnough is called from create.go. With this setup, we check cluster versions before calling EnsureIngress().

Reply:
We should do complete validation first. So validations.Validate() should also be called from create.go before calling EnsureIngress()?

Reply:
It looks like it works okay to move all the validation code into Validate and calling it earlier in create.go. I'm having a little trouble moving the unit tests over cleanly. (will followup on the unit tests...)